### DISCUSSION TITLE
[Feature Request] Option to make manager sidebar panels hover over web content instead of shifting it.

### DISCUSSION THREAD

**Discussion by Crunchbits** [Floorp-Projects/Floorp#419] [Feature Request]
### Is your feature request related to a problem?

The manager sidebar panels currently shift/move web content which can be irritating to see web content constantly moving plus you have to wait for it to readjust every time before using it. Note I know this is already an option for the sidebar, but I am talking about the **manager** sidebar.

### Describe the solution you'd like

Would be great to have an option make it so the panel's hover over the web content.

**Comment by Ryosuke-Asano** [comment]
I have received this request several times, but it is difficult and unlikely to be implemented
